modules/mpv-player/android/src/main/java/expo/modules/mpvplayer/nativeplayer/MediaSessionController.kt

```
@@ -0,0 +1,255 @@
+                    val path = Uri.parse(artworkUri).path ?: artworkUri.removePrefix("file://")
+                    BitmapFactory.decodeFile(File(path).absolutePath)
+                } else {
+                    val url = URL(artworkUri)
+                    url.openStream().use { BitmapFactory.decodeStream(it) }
+                }
+                if (bitmap != null) {
+                    baseBuilder.putBitmap(MediaMetadata.METADATA_KEY_ALBUM_ART, bitmap)
+                    baseBuilder.putBitmap(MediaMetadata.METADATA_KEY_ART, bitmap)
+                    mediaSession?.setMetadata(baseBuilder.build())
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)